Dockerfile

```diff
@@ -0,0 +1,397 @@
+FROM debian:stretch
+
+SHELL ["/bin/bash", "-c"]
+
+WORKDIR /tmp
+
+# Build tools, and misc supporting tools
+RUN apt update && \
+    apt install -y build-essential gfortran automake bison flex libtool git \
+    wget unzip doxygen
+
+# Python v3.7.6
+RUN apt install -y zlib1g-dev libffi-dev libssl-dev && \
+    wget https://www.python.org/ftp/python/3.7.6/Python-3.7.6.tar.xz && \
+    tar -xf Python-3.7.6.tar.xz && rm Python-3.7.6.tar.xz && \
+    mkdir /tmp/Python-3.7.6/build && cd /tmp/Python-3.7.6/build && \
+    ../configure --enable-shared \ 
+    --enable-unicode=ucs4 --enable-ipv6 && \
+    make -j $(nproc --ignore=2) && \
```

Comment:
I see that a lot of the `nproc`-invocations get `--ignore=2`.
Is there any reason not to build faster, or would it make sense to turn this into a non-hardcoded configuration?
As you've commented elsewhere (end of readme) it can take up to several hours to build the image, so for dedicated build computers or when you go for a walk during the build it might be nice to build as fast as possible.